### PR TITLE
Refactoring + Node Size + Source Inspector + Module Global Usage

### PR DESCRIPTION
**Pull request by EtDu**
- [x] Break out the packages list into its own component, DepList
- [x] Move functions from DepGraph into a util file 
- [x] Resize nodes based on file size
 - [x] Add source inspector for modules
- [x] Highlight global usage in source
- [x] Display accurate global usage of modules
- [x] Display correct color for module nodes
- [x] Break out sesifyMode state and Nav component from App, move to DepGraph

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,9 +6,11 @@
   "dependencies": {
     "@babel/preset-env": "^7.8.7",
     "@babel/preset-react": "^7.8.3",
+    "codemirror": "^5.52.2",
```
Comment (kumavis) [inline package.json:9]: do we need this if we have the react component?

File: src/graphs/utils/utils.js [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+  const packageData = {}
+  // create a fake `bundleData` using the packages
+  Object.entries(bundleData).forEach(([parentId, moduleData]) => {
+    const packageName = getPackageVersionName(moduleData)
+    let pack = packageData[packageName]
+    // if first moduleData in package, initialize with moduleData
+    if (!pack) {
+      pack = Object.assign({}, moduleData)
+      pack.file = packageName
+      pack.entry = (packageName === '<root>')
```
Comment (kumavis) [inline src/graphs/utils/utils.js:14]: hmm

what do we use the `entry` property for?
technically you can have an "entry" module (run on init) thats not in the root package

File: src/graphs/utils/utils.js [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+      source,
+      packageName
+    } = parentData
+    const packageVersionName = getPackageVersionName(parentData)
+    //work here
+    const size = getNodeSize(source)
+    const configForPackage = configData.resources[packageName] || {}
+    const configLabel = JSON.stringify(configForPackage, null, 2)
+    const label = `${file}`
+    const isEntryPackage = packageVersionName === '<root>'
```
Comment (kumavis) [inline src/graphs/utils/utils.js:50]: same as above

File: package.json (hunk 1 above)
Comment (EtDu) [reply package.json:9]: Yeah, it's dependent

File: src/graphs/utils/utils.js (hunk 2 above)
Comment (EtDu) [reply src/graphs/utils/utils.js:14]: It appears we don't use the `entry` property anywhere, so it's redundant to set it here.

File: src/graphs/utils/utils.js (hunk 3 above)
Comment (EtDu) [reply src/graphs/utils/utils.js:50]: `isEntryForPackage` is so the <root> node is colored properly

File: src/graphs/utils/utils.js (hunk 2 above)
Comment (kumavis) [reply src/graphs/utils/utils.js:14]: please remove it

**Pull request merged**